python/lsst/pipe/tasks/makeCoaddTempExp.py

```diff
@@ -538,3 +553,194 @@ def applySkyCorr(self, dataRef, calexp):
         if isinstance(calexp, afwImage.Exposure):
             calexp = calexp.getMaskedImage()
         calexp -= bg.getImage()
+
+
+class MakeWarpConfig(pipeBase.PipelineTaskConfig, MakeCoaddTempExpConfig):
+    calExpList = pipeBase.InputDatasetField(
+        doc="Exposures which are resampled and optionally PSF-Matched onto SkyMap projection and pixelation",
+        name="calexp",
+        storageClass="ExposureF",
+        dimensions=("Visit", "Detector")
+    )
+    backgroundList = pipeBase.InputDatasetField(
+        doc="Backgrounds to optionally add back into the calexp",
+        name="calexpBackground",
+        storageClass="Background",
+        dimensions=("Visit", "Detector")
+    )
+    skyCorrList = pipeBase.InputDatasetField(
+        doc="SkyCorr",
+        name="skyCorr",
+        storageClass="Background",
+        dimensions=("Visit", "Detector")
+    )
+    jointcal_wcs = pipeBase.InputDatasetField(
+        doc=("WCS from jointcal or meas_mosaic Placeholder. Pre-flight gets angry about the tract."
+             "Not tested because ci_hsc does not run jointcal or meas_mosaic."),
+        name="jointcal_wcs",
+        storageClass="FitsCatalogStorage",
+        dimensions=("Visit", "Detector, Tract")
+    )
+    jointcal_photoCalib = pipeBase.InputDatasetField(
```

Comment:
Since these datatypes were a placeholder anyway, I'm dropping them and punting until DM-17062.  Outstanding problem the check that `doApplyUberCal` is False is in `validate` (the normal place to check).   Pipelinetask validating and freezing configs is high on my priority list, as we talked about yesterday.